airbyte-workers/src/test/java/io/airbyte/workers/protocols/airbyte/AirbyteMessageTrackerTest.java

```diff
@@ -278,4 +301,13 @@ private AirbyteMessage createStateMessage(final int stateData) {
         .withState(new AirbyteStateMessage().withData(Jsons.jsonNode(stateData)));
   }
 
+  private AirbyteMessage createTraceMessage(final String message, final Long emittedAt) {
+    return new AirbyteMessage()
+        .withType(AirbyteMessage.Type.TRACE)
+        .withTrace(new AirbyteTraceMessage()
+            .withType(AirbyteTraceMessage.Type.ERROR)
+            .withEmittedAt(emittedAt)
+            .withError(new AirbyteErrorTraceMessage().withMessage(message)));
```

Comment:
the reason I left this here is because of the two other methods above it (createRecordMessage and createStateMessage). this one is actually different from the AirbyteMessageUtils method because that one is returning an AirbyteTraceMessage and this one is returning an AirbyteMessage. i did think about moving them all into AirbyteMessageUtils though - do you think that would be best?

Reply:
Yeah I think it would be preferable to move all of these methods into AirbyteMessageUtils, since they are all doing similar things